spec/models/equipment_model_spec.rb

```diff
@@ -286,41 +331,66 @@
 
     context 'methods involving reservations' do
```

Comment:
@esoterik Should I be reviewing the rest of this file or is it still in progress?